site/content/languages/go.md

```diff
@@ -298,15 +298,56 @@ like wazero. In other words, go can't run the wasm it just built. Instead,
 
 Now, you should be all set and can iterate similar to normal Go development.
 The main thing to keep in mind is where files are, and remember to set
-`GOOS=js GOARCH=wasm` when running go commands.
+`GOARCH=wasm GOOS=js` when running go commands.
 
 For example, if you fixed something in the `syscall/js` package
 (`${GOROOT}/src/syscall/js`), test it like so:
 ```bash
-$ GOOS=js GOARCH=wasm go test syscall/js
+$ GOARCH=wasm GOOS=js go test syscall/js
 ok  	syscall/js	1.093s
 ```
 
+### Notes
+
+Here are some notes about testing `GOARCH=wasm GOOS=js`
+
+#### Skipped tests
+
+You may find tests are skipped (e.g. when run with `-v` arg).
+```
+=== RUN   TestSeekError
+    os_test.go:1598: skipping test on js
+```
+
+The go test tree has functions to check if a platform is supported before
+proceeding. This allows incremental development of platforms, or avoids things
+like launching subprocesses on wasm, which won't likely ever support that.
+
+#### Filesystem access
+
+`TestStat` tries to read `/etc/passwd` due to a [runtime.GOOS default][21].
+As `GOARCH=wasm GOOS=js` is a virtualized operating system, this may not make
+sense, because it has no files representing an operating system.
+
+Moreover, as of Go 1.19, tests don't pass through any configuration to hint at
+the real OS underneath the VM. You might suspect running wasm tests on Windows
+would fail, as that OS has no `/etc/passwd` file. In fact, they would except
+Windows tests don't pass anyway because the script that invokes Node.JS,
+[wasm_exec_node.js][22], doesn't actually work on Windows.
+
+```bash
+$ GOOS=js GOARCH=wasm go test os
+fork/exec C:\Users\fernc\AppData\Local\Temp\go-build2236168911\b001\os.test: %1 is not a valid Win32 application.
+FAIL    os      0.034s
+FAIL
+```
+
+Hosts like OS/x and Linux pass these tests because they include files like
```

Comment:
nit: OS/x is not common, and should be Darwin as we are referencing Linux (kernel)

Reply:
yep safe to assume people who are reading this should know OS/x is darwin